core/src/main/java/io/aiven/klaw/controller/TemplateMapController.java

```diff
@@ -378,6 +378,15 @@ public String synchronizeTopics(
     return checkAuth("synchronizeTopics.html", request, response, abstractAuthenticationToken);
   }
 
+  @RequestMapping(value = "/synchronizeSchemas", method = RequestMethod.GET)
+  public String synchronizeSchemas(
```

Comment:
It'd be nice if this was "/sync/schemas"

Reply:
It's hard to change in TemplateMapController